src/build.d

```diff
@@ -776,8 +767,8 @@ auto sourceFiles()
         backend:
             dirEntries(env["C"], "*.d", SpanMode.shallow)
                 .map!(e => e.name)
-                .filter!(e => !e.canFind("dt.d", "obj.d"))
-                .array ~ buildPath(env["C"], "elfobj.d"),
+                .filter!(e => e.baseName != "dt.d" && e.baseName != "obj.d" && e.baseName != "optabgen.d")
```

Comment:
```suggestion
                .filter!(e => e.baseName.among("dt.d", "obj.d", "optabgen.d"))
```

Reply:
Should that be `!e.baseName.among`?

Reply:
I'll test locally and updat the PR.  Thanks.